AEPServices/Sources/ui/alert/AlertMessage.swift

```diff
@@ -0,0 +1,126 @@
+/*
+ Copyright 2020 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+import UIKit
+import WebKit
+
+/// Tthis class is used to create and display alert messages with custom title, message and buttons on the current view
+@objc(AEPAlertMessage)
```

Comment:
Minor suggestion: One thing you could do to reduce the use of `ServiceProvider.shared.messageMonitorService` since it's a long line.

Is declare a computed variable
```swift
private var messageService: MessageMonitorService {
   return ServiceProvider.shared.messageMonitorService
}
```

Then you can just use `messageService` as a normal variable while always getting it from the service provider. You can do this in the full-screen message class too if you'd like.